R/available.R

```diff
@@ -65,7 +65,11 @@ create <- function(name, ...) {
 #' @export
 suggest <- function(path = ".", title = NULL) {
   if (is.null(title)) {
-    title <- unname(desc::desc(path)$get("Title"))
```

Comment:
yeah that's an even better idea. Without `tryCatch`:
```
> suggest("/path/that/exists/")
Error: No root directory found in /path/that/exists or its parent directories [...]
```
with `tryCatch()`:
```
> suggest("/path/that/exists")
Error: No title found, please specify one with `title`.
```
I'll leave it to you to maybe tweak the error message - "one" is somewhat ambiguous there.